src/html-duration-picker.js

```diff
@@ -199,12 +189,8 @@ export default (function() {
       const totalPickerWidth = currentPickerStyle.width;
       picker.setAttribute('data-upgraded', true);
       picker.value = picker.dataset.duration && checkDuration(picker) ? picker.dataset.duration : '00:00:00';
-      picker.style.textAlign = 'right';
-      picker.style.paddingRight = '20px';
-      picker.style.boxSizing = 'border-box';
-      picker.style.width = '100%';
-      picker.style.margin = 0;
-      picker.style.cursor = 'text';
+      picker.setAttribute('style', `text-align:right;padding-right:20px;box-sizing:border-box;
```

Comment:
Using `setAttribute` presents a problem of overriding existing styles on the element. See: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/style